chart/compass/values.yaml

```diff
@@ -2,12 +2,15 @@ global:
   images:
     containerRegistry:
       path: eu.gcr.io/kyma-project/incubator
+    connector:
+      dir: pr/
+      version: "PR-32"
```

Comment:
It doesn't work - you have to modify Connector (parametrize Playground API endpoint), similarly to Director. Just copy `main.go` from `director`, adjust it if necessary and it'll work.

Reply:
Fixed